packages/components/src/border-control/border-control/README.md

```diff
@@ -118,38 +114,39 @@ that selects or clears, border color, style, or width.
 
 _Note: the value may be `undefined` if a user clears all border properties._
 
-- Required: Yes
+-   Required: Yes
 
 ### `shouldSanitizeBorder`: `boolean`
 
 If opted into, sanitizing the border means that if no width or color have been
 selected, the border style is also cleared and `undefined` is returned as the
 new border value.
 
-- Required: No
-- Default: true
+-   Required: No
+-   Default: true
 
 ### `showDropdownHeader`: `boolean`
```

Comment:
Thinking about the API, I wonder if this made sense to be a boolean prop. Just like we have a `label`, it might have made more sense to have a prop that allows controlling that dropdown header label. And if it's specified, we would automatically display the close button. 

@ciampo @mirka WDYT?

Maybe it's a good time to change this if y'all agree. The prop is used only internally in the package.

Reply:
My thoughts:

- If we want to allow customization, we'll need a `colorHeaderLabel` sort of prop.
- I'm not sure I can see a use case for a custom label here though. This is a part of a border control, so while I can see a custom label at the top-level (e.g. "<custom block name or whatever> border") I struggle to find a similar case for this sub-setting.
- Even if we allow customization, it might make sense to have a default. In that case, a prop like this would still make sense.
- A good middle ground could be to keep this `false` by default, but make it `true` by default if the `colorHeaderLabel` prop is set. That way we reduce the chance of accidentally not showing the label even though a custom one has been set.

My conclusion: based on 1. that I don't currently see a use case for a custom label and 2. that even if so, a prop like this would still make sense (with the sensible default heuristic described), I feel like the best path forward is to leave this as-is and worry about it again if/when the use-case for custom label arises, if that ever happens.

Reply:
Alternative suggestion: just show the header at all times and remove this setting.

Reply:
Keeping the default behavior sounds like a good alternative to me. But isn't the default behavior to NOT show the header ATM?

Reply:
> Keeping the default behavior sounds like a good alternative to me. But isn't the default behavior to NOT show the header ATM?

I'm a bit lost, what do you mean here? Just removing the option altogether?

Reply:
Yes. If we never show that label, removing the option does make sense.

Reply:
That'd be one more option to take here. Doing a quick lookup, it doesn't seem like it's ever been used at all.

Reply:
Origin:

> I've just added a prop to toggle its display for now. This is included in the Storybook example controls so it can be evaluated.

Source: https://github.com/WordPress/gutenberg/pull/37769#issuecomment-1074809008

Looping @jasmussen and @WordPress/gutenberg-design for feedback, but it seems like no heading became the de-facto design and we should be able to remove it altogether.

Reply:
I don't see any benefit to making this optional—it only opens the door to inconsistencies.

Whether to display the header/close button or not; we're already a bit inconsistent about this detail across the various popovers in Gutenberg, so I don't have a strong opinion. But generally I'd agree that maintaining the current design makes the most sense for now. 

We can revisit all popovers holistically once there is consensus in https://github.com/WordPress/gutenberg/issues/63674.

Reply:
Thanks for verifying @jameskoster! 

If the current design is to always hide it, and we never use it, I'd just suggest removing the prop altogether.

Reply:
Not just that, we should remove the heading itself too. Will do very soon.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/65475/commits/d357efda081be89758b69f2eefc0f27a68127670

Reply:
Sorry if I'm late to the party, but by removing the type def, we may introduce breaking changes to folks specifying that prop (they may get TS errors). A "softer" alternative would be to keep the type def, adding the `@deprecated` JSDoc tag to deprecated it, and the `@ignore` JSDoc tag to exclude it from JSDocs snippets and Storybook.

cc @mirka to double-check this is a good approach.

Reply:
What do you mean by "folks specifying that prop"? In Gutenberg it's not used anywhere like the prop name string literally doesn't exist anywhere else. Do you mean external consumers of the component?

Reply:
Yes, I mean external consumers. Keeping the typedef around doesn't cost us much, and prevents folks from dealing with build errors.

Reply:
Agreed with Marco's suggestion. And this is technically a breaking change for extenders, but I'm ok with silently removing the feature since it won't cause any functional harm.

Reply:
Alright, will do. Gonna add a deprecation warning though, so that we can remove it at some point in the future.